src/test/java/uk/ac/ebi/eva/pipeline/jobs/GenotypedVcfJobTest.java

```diff
@@ -163,7 +164,6 @@ public void aggregationIsNotAllowed() throws Exception {
                 .toJobParameters();
         JobExecution jobExecution = jobLauncherTestUtils.launchJob(jobParameters);
 
-        assertEquals(ExitStatus.FAILED, jobExecution.getExitStatus());
-        assertEquals(BatchStatus.FAILED, jobExecution.getStatus());
+        assertFailed(jobExecution);
```

Comment:
Be consistent in the way this is called: right above the `JobTestUtils` class is referenced.